cmd/create/oidcconfig/cmd.go

```diff
@@ -101,6 +105,20 @@ func init() {
 		"Prefix for the OIDC configuration, secret and provider.",
 	)
 
+	flags.BoolVar(
+		&args.managed,
+		managedFlag,
+		false,
+		"Indicates whether it is a Red Hat managed or unmanaged (Customer's hosted) OIDC Configuration.",
```

Comment:
NP: `(Customer's hosted)` -- should this be `Customer hosted`? Feels a bit out of place having 's here

Reply:
True